### PR TITLE
Change default value for editor version

### PR DESCRIPTION
**Pull request by maestromac**
## What type of PR is this? (check all applicable)
- [x] Feature/Chore

## Description
Changing default editor from v1 to v2 and removing all the manual code we've sprinkled around

## Related Tickets & Documents
Initially suggested by @citizen428 on https://github.com/forem/forem/pull/13903/files#r644458949

## QA Instructions, Screenshots, Recordings
You should not notice anything with this change

## Added tests?
- [x] adjusted existing specs

## Added to documentation?
- [x] no documentation needed

## [optional] Are there any post deployment tasks we need to perform?
n/a

### CODE REVIEW

**Comment by citizen428** [comment]
Thanks for taking this on @maestromac, judging by the code changes you have so far I think this is the right call.

**Review by juliannatetreault — APPROVED** [review]
Tested locally! 🚀

File: spec/system/user_uses_the_editor_spec.rb [hunk 1]
```diff
@@ -57,7 +57,9 @@ def fill_markdown_with(content)
     end
   end
 
-  describe "Submitting an article", js: true do
+  describe "Submitting an article with v1 editor", js: true do
+    before { user.update(editor_version: "v1") }
```
Comment (jgaskins) [inline spec/system/user_uses_the_editor_spec.rb:61]: We want to be notified if this fails 🙂

```suggestion
    before { user.update!(editor_version: "v1") }
```

**Review by jgaskins — COMMENTED** [review]
This is cool! Just a quick touch up requested for the spec.

**Review by citizen428 — APPROVED** [review]

**Pull request merged**